### ISSUE TITLE
[Manageable Affordances TF] Workflow for CG Report

### ISSUE THREAD

**Issue by egekorkan** [w3c-cg/webagents#37]
We have collected a good amount of use cases from various participants. The end goal is to turn them into a CG report. For this, with @andreiciortea we thought of a simple process:

1. Get motivating scenarios via GitHub issues
2. Extract the requirements together with the submitter. The list of requirements are at https://github.com/w3c-cg/webagents/issues/34
3. Identify the gaps in the standards (WoT, other W3C, other SDOs) and open specifications (e.g. Hydra, JSON-HAL)
4. (Optional) Propose a concrete mechanism to satisfy the gap. This can be some diagram that gives enough information to implement or can be a proof of concept implementation

Regarding the mechanical workflow:

1. Description and requirements are collected in the issue. The issue owner can edit and extend their first comment with further information.
2. Once there are enough requirements, the moderators or the issue creator can open a Pull Request to https://github.com/w3c-cg/webagents/tree/main/TaskForces/ManageableAffordance/Submissions by copy-pasting their first comment contents in markdown. The file should be named `{issue number}-{title of the scenario}.md` to avoid name conflicts and track the submission back to the issue.
3. A wider group extends the submission with the gap analysis as a follow-up Pull Request. A conversation can happen in the Pull Request
4. Once the Pull Request is merged, the group can turn it into a section in the report HTML document.

Once we have enough contributions to the HTML and it is reviewed, it can be published as a CG report.
